modules/ROOT/pages/enabling-api-catalog.adoc

```diff
@@ -0,0 +1,33 @@
+= Enabling API Catalog
+
+Enable a connected Salesforce organization to access and sync Anypoint Platform APIs into the Salesforce API Catalog and expose them as invocable actions. This capability is enabled in Anypoint Platform Access Management by an Anypoint Platform organization administrator.  
+
+[[enable-disable-api-catalog]]
+== Enable or Disable API Catalog in Anypoint Platform
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. In *Capabilities Managed in Anypoint Platform*, click *Settings* for *Make MuleSoft Anypoint Platform APIs Available in Salesforce*.
+. In *Settings for Anypoint APIs in Salesforce*, click *Continue*.
+. In *Settings*, select the Anypoint Platform business groups that allow the Salesforce organization to import Anypoint Platform APIs into its API Catalog, and click *Save Settings*.
++
+[NOTE]
+Within the selected business groups, you can import only APIs that have specs published in Exchange with active instances into Salesforce.
+. In *Capabilities Managed in Anypoint Platform*, switch between *Enabled* or *Disabled* in *Make MuleSoft Anypoint Platform APIs Available in Salesforce*. 
++
+image:connected-salesforce-org-api-catalog-enabled.png["Connected Salesforce organization with both Invocable Actions and API Catalog enabled"]
++
+The Salesforce organization administrator can then sync APIs into their catalog. 
++
+For details about syncing APIs and using API Catalog in Salesforce, see xref:general::api-catalog-in-salesforce.adoc[].
```

Comment:
Will this link be replaced with a link to the topic in Salesforce Help? I have the planned link, but need to get the topic ID approved by a content strategist before I share it.